infra/terraform/external_function/snowalert.tf

```diff
@@ -2,37 +2,98 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.0"
+      version = "3.34.0"
     }
+    
+    # snowflake = {

```

Comment:
Are we not using the `snowflake` provider in this module?

Reply:
we are using it but it is defined in api_integration.tf